server/api/batch_tallies.py

```diff
@@ -311,3 +320,57 @@ def download_batch_tallies_template(
         string_io,
         filename=f"candidate-totals-by-batch-template-{jurisdiction_timestamp_name(election, jurisdiction)}.csv",
     )
+
+
+@api.route(
+    "/election/<election_id>/batch-tallies/summed-by-jurisdiction-csv", methods=["GET"],
+)
+@restrict_access([UserType.AUDIT_ADMIN, UserType.JURISDICTION_ADMIN])
```

Comment:
Can jurisdiction admins download this file? I don't think so. So I think it makes sense to remove their access